ISS/rforms.py

```diff
@@ -0,0 +1,26 @@
+from django import forms            
+from django.contrib.auth.models import User   
+from django.contrib.auth.forms import UserCreationForm      
+
+class MyRegistrationForm(UserCreationForm):
+    email = forms.EmailField(required = True)
+    first_name = forms.CharField(required = False)
+    last_name = forms.CharField(required = False)
+    birtday = forms.DateField(required = False)
+
+    class Meta:
+        model = User
```

Comment:
Like the use for a form for registration here although we use `Poster` as our user model rather than auth's `User` model, so that will have to be reflected here. Also for that reason there are no first/last names and birthday fields, just a `username` field.